libminifi/test/unit/PropertyTests.cpp

```diff
@@ -22,6 +22,39 @@
 #include "../../include/core/Property.h"
 #include "utils/StringUtils.h"
 #include "../TestBase.h"
+namespace {
```

Comment:
Do we need the anonymous namespace here?

Reply:
It is a good practice to put all tests into anon namespaces.

Reply:
Thanks, I was just wondering if it was due to a linker error or some other reason.